Makefile

```diff
@@ -28,12 +31,14 @@ $$($(1)_OBJS): $(BUILD)/$(1)/%.o: %.c
 $(BUILD)/$(1).bin $(BUILD)/$(1).elf: $$($(1)_OBJS)
 	$(Q)$(CC) $$($(1)_CFLAGS) $$($(1)_LDFLAGS) $$($(1)_OBJS) -Wl,-T$$($(1)_LDSCRIPT) -o $(BUILD)/$(1).elf
 	$(Q)$(OBJCOPY) -O binary -R .eeprom $(BUILD)/$(1).elf $(BUILD)/$(1).bin
+
 endef
 
 $(foreach t,$(TARGETS),$(eval $(call each_target,$(t))))
 
 clean:
 	@-rm -rf $(BUILD)
+	@-rm version.c
```

Comment:
hmm, how about we put version.c in `$(BUILD)/`? It is a generated file, after all.